tests/backend/modules/test_STT.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from ....CogNative.backend.modules.STT import STT
+
+from nltk.translate import bleu
+from nltk.translate.bleu_score import SmoothingFunction
+
+
+def test_invalid_language():
+    with pytest.raises(Exception):
+        s = STT(source_language="randomness")
+
+def test_english():
+    s = STT(source_language="english")
+    example_path = "tests/examples/BillMaher22.wav"
+    text = s.speech_to_text(example_path)
+
+    actual_text = ("it wasn't really necessary I tried to "
+                   "explain one night on my show that I "
+                   "think a lot of this comes from he thinks"
+                   " he's the savior of the Russian people I"
+                   " think when you get to that level where "
+                   "he's been in office he's been an absolute"
+                   " power for 20 years what is left what "
+                   "you know what's in the soul of a man he "
+                   "wants to be a hero and he thinks this is "
+                   "the way to do it I think people know "
+                   "this is history major stuff")
+    
+    # Check if synthesized text vs actual text is greater than 90%
+    smoothie = SmoothingFunction().method4
+    assert bleu([actual_text], text, smoothing_function=smoothie) > 0.9
+
+    example_path2 = "tests/examples/DavidHowell15.wav"
+    text2 = s.speech_to_text(example_path2)
+
+    actual_text2 = ("Once upon a time in a place not far "
+                    "from where you are standing there lived"
+                    " a community of farmers merchants "
+                    "blacksmiths strong women and a few others"
+                    " who used the resources they had "
+                    "available to make life work")
+    
+    # Check if synthesized text vs actual text is greater than 90%
+    assert bleu([actual_text2], text2, smoothing_function=smoothie) > 0.9
+
+    # Check if get_transcriptions() is storing the correct
+    # correct text with the source audio file
+    assert s.get_transcriptions()[example_path] == text
+    assert s.get_transcriptions()[example_path2] == text2
```

Comment:
```STT.get_transcriptions``` returns a ```dict``` with path's associated with text transcribed from the audio at that path. These assertions are checking that the text is associated with the correct path.